crates/lambda-platform/src/winit/mod.rs

```diff
@@ -66,6 +78,77 @@ pub struct WindowHandle {
   pub monitor_handle: MonitorHandle,
 }
 
+// Should we take the loop as a field right here? Probably a ref or something? IDK
+pub struct WindowHandleBuilder {
+  window_handle: Option<Window>,
+  size: Option<WindowSize>,
```

Comment:
We could most likely specify a default size to avoid making it an optional property.